rebar.lock

```diff
@@ -18,7 +18,7 @@
   0},
  {<<"aeserialization">>,
   {git,"https://github.com/aeternity/aeserialization.git",
-      {ref,"3416ff5c1b4f6401bc211106063717ea1bbb1bd2"}},
+      {ref,"4a072977745571ceb6cf34e6387a96b808ae20b3"}},
```

Comment:
https://github.com/aeternity/aeserialization/compare/3416ff5c1b4f6401bc211106063717ea1bbb1bd2...4a072977745571ceb6cf34e6387a96b808ae20b3
> tag(channel_client_reconnect_tx) -> 575;
> ...

Reply:
Shall this require doc update in https://github.com/aeternity/protocol/blob/master/serializations.md ? 🤔

Reply:
Is this tx anyhow under consensus?

Reply:
> Is this tx anyhow under consensus?

No, like the `aesc_offchain_tx`, it is not designed to go on-chain: the `fee` is hard-coded to zero, as is the `ttl`, and `aesc_offchain_tx:process()` will raise an exception.

Reply:
> Shall this require doc update in https://github.com/aeternity/protocol/blob/master/serializations.md ? thinking

Working on documentation now ([ticket](https://www.pivotaltracker.com/story/show/167617278))